.github/workflows/build-release-binaries.yml

```diff
@@ -138,6 +142,8 @@ jobs:
           COORDINATOR_P2P_ENDPOINT: 022ae8dbec1caa4dac93f07f2ebf5ad7a5dd08d375b79f11095e81b065c2155156@34.30.20.202:9045
           NETWORK: regtest
           COORDINATOR_PORT_HTTP: 80
+          ORACLE_ENDPOINT: http://api.test.10101.finance:8081
+          ORACLE_PUBKEY: 5d12d79f575b8d99523797c46441c0549eb0defb6195fe8a080000cbe3ab3859
```

Comment:
❓Is it on purpose that we specify the public regtest pubkey for android and not the mainnet pubkey (as for ios)?

Reply:
This is building the android release for regtest, hence we use the regtest oracle.

It should be the same for ios. Note you might have mixed it up as we are having an ios regtest and mainnet build.